classes/omnect_uboot_configure_env.bbclass

```diff
@@ -3,7 +3,8 @@ omnect_uboot_configure_env() {
     cp -f ${WORKDIR}/omnect_env.h ${S}/include/configs/
 
     # set omnect_u-boot_version
-    sed -i -e "s|^#define OMNECT_ENV_BOOTLOADER_VERSION$|#define OMNECT_ENV_BOOTLOADER_VERSION \"omnect_u-boot_version=${PKGV}\\\0\"|g" ${S}/include/configs/omnect_env.h
+    OMNECT_BOOTLOADER_VERSION=$(cat ${WORKDIR}/omnect_bootloader_version)
```

Comment:
is this necessary due to 
d.setVar("OMNECT_BOOTLOADER_VERSION", omnect_bootloader_version)

in classes/omnect_bootloader_versioning.bbclass

Reply:
var is set only in context of `do_bootloader_checksum()` ... and there unnecessary. i'll remove it.
it is not set in context of `omnect_uboot_configure_env()`

Reply:
removed `d.setVar("OMNECT_BOOTLOADER_VERSION", omnect_bootloader_version)` in classes/omnect_bootloader_versioning.bbclass